### DISCUSSION TITLE
Unable to add domain

### DISCUSSION THREAD

**Discussion by oluwakeye-john** [vercel/community#6636] [Help]
### Summary

I'm trying to migrate my domain, uselumen.co from Netlify to Vercel, but I'm unable to add the domain on my Vercel account. 

The error on my Vercel dashboard is
```
Another Vercel account is using this domain
```

I'll like to know which account is currently using my domain and how I can transfer my domain back to my Vercel account.

### Steps to Reproduce

<img width="942" alt="image" src="https://github.com/vercel/community/assets/43508135/d65c245c-4379-4ce8-b610-92f630b679fb">

**Comment by amyegan** [comment]
Hey @oluwakeye-john. You're seeing that message because someone else, likely a previous owner of the domain, has already associated the domain with their account. If you [verify domain access](https://vercel.com/docs/projects/domains/add-a-domain#verify-domain-access) by adding that TXT record, then it can be used for your project.

**Comment by oluwakeye-john** [comment]
Thanks for the response @amyegan 

I've verified the domain using the TXT and A records and I can now add the domain to my project. (I'm currently managing the DNS records on Netlify)

However, I'll really like to control the domain exclusively from my personal Vercel account as I exclusively purchased and currently own the domain.

I've encountered this issue with a previous domain at work and it was caused by Mintlify registering the domain for my docs. I messaged Mintlify and they removed the domain from the end and I was able to add the domain to my Vercel account.

In the case of my current domain, I'm not sure of the service using the domain on my behalf as there are a bunch of different services connected to the domain.

I'll be glad to provide extra information if you need. 

Thanks

**Reply by amyegan** [reply]
Thanks for verifying domain ownership. I see that it was already moved to your account and you should be able to see it in your team's domains list now.
